app/messaging/process-message-request.js

```diff
@@ -4,13 +4,16 @@ import { validateMessageRequest } from './message-request-schema.js'
 export const processMessageRequest = async (logger, message, receiver) => {
   try {
     logger.warn(`Received sfd message request with id: ${message.messageId}`)
-    const messageBody = message.body
-    if (validateMessageRequest(logger, messageBody)) {
-      await sendMessageToSingleFrontDoor(logger, message.messageId, messageBody)
-      await receiver.completeMessage(message)
+
+    if (!validateMessageRequest(logger, message.body)) {
+      await receiver.deadLetterMessage(message) // not worth retrying, invalid input
+      return
     }
+
+    await sendMessageToSingleFrontDoor(logger, message.messageId, message.body)
+    await receiver.completeMessage(message)
   } catch (err) {
-    await receiver.deadLetterMessage(message)
-    logger.error(`Unable to process sfd message request: ${err.message}`)
+    logger.error(`Message processing failed with message: ${err.message}`)
```

Comment:
when logging errors (or adding them to bindings) think we should log the full error object over just the message.

Reply:
Agreed, will update now